src/android/plugin/google/maps/PluginMarker.java

```diff
@@ -958,8 +959,6 @@ protected Bitmap doInBackground(Void... params) {
           if (image == null) {
             return null;
           }
-          icons.add(image);
```

Comment:
Deleting this line, you are misunderstand.
Caching local image reference to the icon property is for releasing them on clear method.

```java
    Bitmap[] cachedBitmaps = icons.toArray(new Bitmap[icons.size()]);
    Bitmap image;
    for (int i = 0; i < cachedBitmaps.length; i++) {
      image = icons.remove(0);
      if (image != null && !image.isRecycled()) {
        image.recycle();
      }
      image = null;
    }
```

Deleting this line means local icon images are not released.